File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_ppt_template_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusPptTemplateVO:
    """PPT模板VO对象，存储PPT模板的基本信息及关联关系"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    title: str = None
    description: str = None
    cover_images: ImageMediaResourceList = None
    type: str = None
    status: str = None
    author_id: int = None
    author_name: str = None
    tags: PlusPptTemplateTags = None
    meta: PlusPptTemplateMeta = None
    slides: List[PlusPptTemplateSlideVO] = None

File: sdkwork-backend-sdk-go/types/plus_ppt_template_vo.go
package types

// PPT模板VO对象，存储PPT模板的基本信息及关联关系
type PlusPptTemplateVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	Title string `json:"title"`
	Description string `json:"description"`
	CoverImages ImageMediaResourceList `json:"coverImages"`
	Type string `json:"type"`
	Status string `json:"status"`
	AuthorId int `json:"authorId"`
	AuthorName string `json:"authorName"`
	Tags PlusPptTemplateTags `json:"tags"`
	Meta PlusPptTemplateMeta `json:"meta"`
	Slides []PlusPptTemplateSlideVO `json:"slides"`
}
